Add bookmarks to specific list (#4)
mathpn/hoarder-miniflux-webhook: README.md

   WEBHOOK_SECRET=your_generated_secret
   ```

### 5. Set `LIST_ID` (optional, required if `ADD_TO_LIST` is true)

> You need `curl` and `jq` installed for this step.

Reviewer: Mention that `jq` is not strictly necessary:

```suggestion
You need `curl` installed for this step. Here we use `jq` for code formatting, but it is not strictly required. An alternative is to use `python -m json.tool`.
```

> 1. Run this command:

Reviewer: ```suggestion
1. Make a request to list Hoarder lists:
```

```
curl -H 'Authorization: Bearer <hoarder api key>' -L 'http://<hoarder instance>/api/v1/lists' -H 'Accept: application/json' | jq '.'
```

> You'll get something that looks similar this:

Reviewer: Explain what is the response:
```suggestion
The response should contain all Hoarder lists you've created, like the following example:
```
